sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -267,10 +308,12 @@ export class MessageSender extends LinkEntity {
           };
           return reject(translate(e));
         };
+        const waitTimer = setTimeout(actionAfterTimeout, retryTimeoutInMs);
```

Comment:
Not sure this is a good change - prior to this you only set the timeout if it wasn't open. 

Is there a reason this was needed?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8159#issuecomment-608254608